src/lib/commandBarConfigs/modelingCommandConfig.ts

```diff
@@ -51,8 +51,12 @@ export type ModelingCommandSchema = {
     distance: KclCommandValue
   }
   Sweep: {
+    // Enables editing workflow
+    nodeToEdit?: PathToNode
+    // Arguments
     target: Selections
     trajectory: Selections
+    sectional: boolean
```

Comment:
I guess this should later be turned into a KCL expression? I believe last I tried booleans weren't evaluated in the command bar when the 'kcl' type of arg was set, but I could be mistaking.